<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorControlWidget</class>
 <widget class="QWidget" name="SensorControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>한화 L</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lightImg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lightButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/logoImage/images/Checkbox_Off.png</normaloff>
      <normalon>:/logoImage/images/Checkbox_On.png</normalon>
      <disabledoff>:/logoImage/images/Checkbox_Off.png</disabledoff>
      <disabledon>:/logoImage/images/Checkbox_On.png</disabledon>
      <activeoff>:/logoImage/images/Checkbox_Off.png</activeoff>
      <activeon>:/logoImage/images/Checkbox_On.png</activeon>
      <selectedoff>:/logoImage/images/Checkbox_Off.png</selectedoff>
      <selectedon>:/logoImage/images/Checkbox_On.png</selectedon>:/logoImage/images/Checkbox_Off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="curtainButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/logoImage/images/Checkbox_Off.png</normaloff>
      <normalon>:/logoImage/images/Checkbox_On.png</normalon>
      <disabledoff>:/logoImage/images/Checkbox_Off.png</disabledoff>
      <disabledon>:/logoImage/images/Checkbox_On.png</disabledon>
      <activeoff>:/logoImage/images/Checkbox_Off.png</activeoff>
      <activeon>:/logoImage/images/Checkbox_On.png</activeon>
      <selectedoff>:/logoImage/images/Checkbox_Off.png</selectedoff>
      <selectedon>:/logoImage/images/Checkbox_On.png</selectedon>:/logoImage/images/Checkbox_Off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="curtainImg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="gasButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/logoImage/images/Checkbox_Off.png</normaloff>
      <normalon>:/logoImage/images/Checkbox_On.png</normalon>
      <disabledoff>:/logoImage/images/Checkbox_Off.png</disabledoff>
      <disabledon>:/logoImage/images/Checkbox_On.png</disabledon>
      <activeoff>:/logoImage/images/Checkbox_Off.png</activeoff>
      <activeon>:/logoImage/images/Checkbox_On.png</activeon>
      <selectedoff>:/logoImage/images/Checkbox_Off.png</selectedoff>
      <selectedon>:/logoImage/images/Checkbox_On.png</selectedon>:/logoImage/images/Checkbox_Off.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gasImg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>330</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>한화 L</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>RECORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="cctvImg">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
